hypergan/cli.py

```diff
@@ -141,7 +144,11 @@ def train(self):
                 self.args.save_every > 0 and
                 i % self.args.save_every == 0):
                 print(" |= Saving network")
-                self.gan.save(self.save_file)
+                self.gan.save(self.save_file)   
+                self.create_path(self.advSavePath+'advSave.txt')
+                    with open(self.advSavePath+'advSave.txt', 'w') as the_file:
```

Comment:
I'm getting a syntax error.  Indentation?

Reply:
Maybe...i wrote directly to github,not copied from my code...i have no syntax errors